### PR TITLE
[MERGE] Merge 1.6 to master

### PR DESCRIPTION
**Pull request by aaudiber**

### CODE REVIEW

File: deploy/vagrant/conf/alluxio.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ Github:
   Version: branch-1.6
 
 Release:
-  Version: 1.6.0 # name of release version
+  Version: 1.6.1 # name of release version
```
Comment (madanadit) [inline deploy/vagrant/conf/alluxio.yml]: @aaudiber is this intended?

File: integration/docker/Dockerfile [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 FROM openjdk:8-jdk-alpine
 
-ARG ALLUXIO_TARBALL=http://downloads.alluxio.org/downloads/files/1.6.0/alluxio-1.6.0-bin.tar.gz
+ARG ALLUXIO_TARBALL=http://downloads.alluxio.org/downloads/files/1.6.1/alluxio-1.6.1-bin.tar.gz
```
Comment (madanadit) [inline integration/docker/Dockerfile]: ditto

File: deploy/vagrant/conf/alluxio.yml (hunk 1 above)
Comment (aaudiber) [reply deploy/vagrant/conf/alluxio.yml]: yes, 1.6.1 is the latest release version

File: integration/docker/Dockerfile (hunk 2 above)
Comment (aaudiber) [reply integration/docker/Dockerfile]: yep

**Pull request merged**